widget/accordion.go

```diff
@@ -232,12 +232,12 @@ func (r *accordionRenderer) updateObjects() {
 		r.headers[i].Hide()
 	}
 	// Set objects
-	var objects []fyne.CanvasObject
-	for _, h := range r.headers {
-		objects = append(objects, h)
+	objects := make([]fyne.CanvasObject, len(r.headers)+len(r.container.Items))
+	for i, header := range r.headers {
+		objects[i] = header
 	}
-	for _, i := range r.container.Items {
-		objects = append(objects, i.Detail)
+	for i, item := range r.container.Items {
+		objects[len(r.headers)+i] = item.Detail
```

Comment:
`len(r.headers)` is recalculated for each iteration of the loop, and is calculated on line 235. Instead, consider calculating it once and storing in a variable

Reply:
Good point. Will do that :+1:

Reply:
I fixed it and managed to drop a few more calls to append.